server/models/Team.ts

```diff
@@ -121,16 +121,20 @@ class Team extends ParanoidModel {
   }
 
   get url() {
+    // custom domain
     if (this.domain) {
       return `https://${this.domain}`;
     }
 
-    if (!this.subdomain || process.env.SUBDOMAINS_ENABLED !== "true") {
-      return process.env.URL;
+    if (!this.subdomain || env.SUBDOMAINS_ENABLED !== "true") {
+      return env.URL;
     }
 
-    const url = new URL(process.env.URL || "");
-    url.host = `${this.subdomain}.${stripSubdomain(url.host)}`;
+    // TODO: HACK: new URL("") will throw an exception
+    // this is just to make the typing work but reall we should
+    // guarantee the existence of env.URL
+    const url = new URL(env.URL || "");
```

Comment:
I assume this should be squared away as soon as the new env var stuff is merged. Leaving this big ugly comment here until that's done